internal/testing/e2e/in_memory.go

```diff
@@ -123,3 +146,32 @@ func createMinimatchForTest(t *testing.T) *rpcTesting.TestContext {
 	tc.AddCloseFunc(closer)
 	return tc
 }
+
+// Create a mmf service using a started test server.
+// Inject the port config of mmlogic using that the passed in test server
+func createMatchFunctionForTest(t *testing.T, c *rpcTesting.TestContext) *rpcTesting.TestContext {
+	// TODO: Use insecure for now since minimatch and mmf only works with the same secure mode
+	tc := rpcTesting.MustServeInsecure(t, func(p *rpc.ServerParams) {
+		cfg := viper.New()
+
+		// The below configuration is used by GRPC harness to create an mmlogic client to query tickets.
+		cfg.Set("api.mmlogic.hostname", c.GetHostname())
+		cfg.Set("api.mmlogic.grpcport", c.GetGRPCPort())
+		cfg.Set("api.mmlogic.httpport", c.GetHTTPPort())
+
+		assert.Nil(t, mmfHarness.BindService(p, cfg, &mmfHarness.FunctionSettings{
+			Func: pool.MakeMatches,
+		}))
+	})
+	return tc
+}
+
+// Create an evaluator service that will be used by the minimatch tests.
+func createEvaluatorForTest(t *testing.T) *rpcTesting.TestContext {
```

Comment:
Shouldnt we be setting the ports that the evaluator got started on in createMinimatchForTest for the synchronizer to know where to find the evaluator? Like the below:
	cfg.Set("api.synchronizer.hostname", tc.GetHostname())
	cfg.Set("api.synchronizer.grpcport", tc.GetGRPCPort())
	cfg.Set("api.synchronizer.httpport", tc.GetHTTPPort())